doc/dev/osd_internals/multi_object_write_transaction.rst

```diff
@@ -0,0 +1,113 @@
+==============================
```

Comment:
@dragonylffly Please prepend doc before PR title.
`It is required to prefix it with the subsystem or module you are changing`
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits